DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/vlan_utils.py

```diff
@@ -0,0 +1,51 @@
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+
+async def configure_bridge_setup(device, dut_ports, default_pvid=0):
+    device.applog.info("Creating bridge entity and enslaving links")
+    out = await IpLink.add(input_data=[
+        {device: [{"device": "br0", "type": "bridge", "vlan_filtering": 1, "vlan_default_pvid": default_pvid}]}])
+    assert out[0][device]["rc"] == 0, f"Failed creating bridge."
+
+    await IpLink.set(input_data=[{device: [{"device": "br0", "operstate": "up"}]}])
+    assert out[0][device]["rc"] == 0, f"Failed setting bridge to state UP."
+
+    out = await IpLink.set(
+        input_data=[{device: [{"device": port, "operstate": "up", "master": "br0"} for port in dut_ports]}])
+    assert out[0][device]["rc"] == 0, f"Failed setting link to state UP."
+
+
+async def configure_vlan_setup(device, port_vlan_map, dut_ports):
+    device.applog.info("Setting links to VLANS")
+    for config in port_vlan_map:
+        port = dut_ports[config["port"]]
+        out = await BridgeVlan.add(input_data=[{device: [
+            {"device": port, "vid": setting["vlan"], "pvid": setting["pvid"], "untagged": setting["untagged"]} for
+            setting in config["settings"]]}])
+        assert out[0][device]["rc"] == 0, f'Vlan: {config["settings"]["vlan"]}  is not added to link: {port}.'
+
+
+def get_traffic_port_vlan_mapping(streams, port_vlan_map, tg_ports, tx_port=0, default_pvid=0):
+    """
+    - Creates traffic-port  dict that can be used  to verify traffic on ports
```

Comment:
Please add a more descriptive explanation. That this is a map of traffic item names to **rx** ports.